src/components/Birds.js

```diff
@@ -24,7 +24,6 @@ const Birds = () => {
         <div className="wrapper">
           <img src={mountainImage3} alt="Mountain image"></img>
         </div>
-
```

Comment:
why this line has been removed, avoid unnecessary changes, check files before you push changes